src/components/Dropdown/Dropdown.jsx

```diff
@@ -615,6 +617,10 @@ Dropdown.propTypes = {
    * For display the drop down menu in overflow hidden/scroll container which contains transform css function usage.
    */
   insideOverflowWithTransformContainer: PropTypes.bool,
+  /**
+   * For when insideOverflowWithTransformContainer is used, this is the ref of the container dialog.
+   */
+  containerRef: PropTypes.func,
```

Comment:
Thanks for the suggestions!
I pushed a fix to independently find the scrollable parent, without passing the `containerRef`.
I had 2nd thoughts after we spoke and I think we might not want to allow user to "stick" the menu to things other than the dropdown:
1. It might break how we meant the component to render
2. There's not necessarily a use case where user would like to "stick" it to stuff other than the dropdown

With saying that, we should really consider - on another dropdown iteration - making this behaviors by default without hacks as this prop. I believe this is something that should be achievable